lib/Factorial.js

```diff
@@ -26,13 +26,18 @@ class Factorial extends Expression
     {
         scope = defaultScope.buildDefaultScope(scope);
 
-        this.value = 1;
-
         this.content = this.content.eval(scope, depth + 1);
 
+        if(this.content.value > 170 || this.content.value < 0) 
+        { 
+            throw new RangeError('Factorial content out of range, must be 0 - [-]170 inclusive.');
```

Comment:
What about rewording this to be `'Factorial content out of range, must be between 0 and 170, inclusive.'`? It's a bit more clear, IMHO.

Reply:
Done.